packages/theming/theming-utils/src/mapPipelineToShadow.ts

```diff
@@ -2,22 +2,22 @@ import { ThemeShadowDefinition } from '@fluentui-react-native/theme-types/src/Sh
+ * @param pipelineOutputShadow Assumes that this is the object in the tokens-shadow.json file of the pipeline output
  * @returns Object containing shadow tokens
  */
 export function mapPipelineToShadow(pipelineOutputShadow: any): ThemeShadowDefinition {
   return {
-    shadow2: { ambient: pipelineOutputShadow[2][0], key: pipelineOutputShadow[2][1] },
-    shadow4: { ambient: pipelineOutputShadow[4][0], key: pipelineOutputShadow[4][1] },
-    shadow8: { ambient: pipelineOutputShadow[8][0], key: pipelineOutputShadow[8][1] },
-    shadow16: { ambient: pipelineOutputShadow[16][0], key: pipelineOutputShadow[16][1] },
-    shadow28: { ambient: pipelineOutputShadow[28][0], key: pipelineOutputShadow[28][1] },
-    shadow64: { ambient: pipelineOutputShadow[64][0], key: pipelineOutputShadow[64][1] },
-    shadow2brand: { ambient: pipelineOutputShadow.brand[2][0], key: pipelineOutputShadow.brand[2][1] },
-    shadow4brand: { ambient: pipelineOutputShadow.brand[4][0], key: pipelineOutputShadow.brand[4][1] },
-    shadow8brand: { ambient: pipelineOutputShadow.brand[8][0], key: pipelineOutputShadow.brand[8][1] },
-    shadow16brand: { ambient: pipelineOutputShadow.brand[16][0], key: pipelineOutputShadow.brand[16][1] },
-    shadow28brand: { ambient: pipelineOutputShadow.brand[28][0], key: pipelineOutputShadow.brand[28][1] },
-    shadow64brand: { ambient: pipelineOutputShadow.brand[64][0], key: pipelineOutputShadow.brand[64][1] },
+    shadow2: { ambient: pipelineOutputShadow.global.shadow2[0], key: pipelineOutputShadow.global.shadow2[1] },
+    shadow4: { ambient: pipelineOutputShadow.global.shadow4[0], key: pipelineOutputShadow.global.shadow4[1] },
+    shadow8: { ambient: pipelineOutputShadow.global.shadow8[0], key: pipelineOutputShadow.global.shadow8[1] },
+    shadow16: { ambient: pipelineOutputShadow.global.shadow16[0], key: pipelineOutputShadow.global.shadow16[1] },
+    shadow28: { ambient: pipelineOutputShadow.global.shadow28[0], key: pipelineOutputShadow.global.shadow28[1] },
+    shadow64: { ambient: pipelineOutputShadow.global.shadow64[0], key: pipelineOutputShadow.global.shadow64[1] },
+    shadow2brand: { ambient: pipelineOutputShadow.global.shadowBrand2[0], key: pipelineOutputShadow.global.shadowBrand2[1] },
+    shadow4brand: { ambient: pipelineOutputShadow.global.shadowBrand4[0], key: pipelineOutputShadow.global.shadowBrand4[1] },
+    shadow8brand: { ambient: pipelineOutputShadow.global.shadowBrand8[0], key: pipelineOutputShadow.global.shadowBrand8[1] },
+    shadow16brand: { ambient: pipelineOutputShadow.global.shadowBrand16[0], key: pipelineOutputShadow.global.shadowBrand16[1] },
+    shadow28brand: { ambient: pipelineOutputShadow.global.shadowBrand28[0], key: pipelineOutputShadow.global.shadowBrand28[1] },
+    shadow64brand: { ambient: pipelineOutputShadow.global.shadowBrand64[0], key: pipelineOutputShadow.global.shadowBrand64[1] },
```

Comment:
@TravisSpomer is this change intentional? Seems weird to see `.global` for the shadow alias tokens.

Reply:
I see. So Shadows are in a weird "global but not global state". Can we...snap it to one or the other? 😅 In the short term, I guess it's OK to have the namespace here be weird since people who use FURN won't need to touch the names

Reply:
Conceptually they're global tokens because they're building blocks with no semantic meaning. Technically they're alias tokens because they refer to other tokens (colors). So they're... global alias tokens. I think we're kind of stuck with them being a weird case.